server/packaging/module_skeleton/lib/puppet/face/language_server/quickstart.rb

```diff
@@ -0,0 +1,32 @@
+require 'puppet/face'
+
+Puppet::Face.define(:language_server, '0.1.0') do
+  action :quickstart do
+    summary "Start a Puppet Language Server with quick defaults"
+    description <<-EOT
+      Starts a puppet language server with quick default settings:
+        - Port 8080
```

Comment:
This conflicts with the help in [`start.rb`](https://github.com/glennsarti/puppet-vscode/blob/84e7d0a0beb75a6d3c4377c552992ddf32736eb6/server/packaging/module_skeleton/lib/puppet/face/language_server/start.rb#L12-L15) where the default port is defined as `8081`.